app/controllers/blogs_controller.rb

```diff
@@ -1,73 +1,89 @@
 class BlogsController < ApplicationController 
-    def index
-        #@user = User.find_by(id:params[:user_id])
-        if(session[:user_id])
-            @user = User.find_by(id:session[:user_id])
-            @blog = @user.blogs.all
-        end
+  def all_blogs
+    @blog = Blog.all
```

Comment:
use `@blogs ||= Blog.all`
1. proper naming should be done. blog means singular, blogs will be plural
2. figure out why `||` needs to be used